tests/UserRepository-test.js

```diff
@@ -0,0 +1,20 @@
+
+const chai = require('chai');
+const expect = chai.expect;
+const activityData = require('../data/activity.js');
```

Comment:
I still want to figure out the difference between ../ and ./ 
I think it might have something to do with which file we were in